### DISCUSSION TITLE
Resnet50 Inference [1, 3, 480, 1024] use 11G GPU Memory

### DISCUSSION THREAD

**Discussion by LossNAN** [open-mmlab/mmclassification#1795] [Q&A]
When I use the ResNet-50 classification network with an image size of [1, 3, 480, 1024] for inference using the eval mode, the GPU memory consumption is close to 11 GB. Is this normal?

```
with torch.no_grad():
    self.model = init_model(config, checkpoint)
    
    tensor = torch.randn(1, 3, 480, 1024).to('cuda:0').half()
    data = {'img': tensor}
    
    for i in range(10):
        _ = self.model(return_loss=False, **data)
```
![image](https://github.com/open-mmlab/mmpretrain/assets/18476895/847f29f4-875b-4be0-9d2f-8f5598cfa201)
